ci-operator/step-registry/ipi/install/workers-rhel7/prep/ipi-install-workers-rhel7-prep-commands.sh

```diff
@@ -0,0 +1,124 @@
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+cat > prep.yaml <<-'EOF'
+---
+- name: Prep Playbook
+  hosts: new_workers
+  any_errors_fatal: true
+  gather_facts: false
+
+  vars:
+    cluster_version: "{{ lookup('env', 'CLUSTER_VERSION') }}"
+    ops_mirror_path: "{{ lookup('env', 'OPS_MIRROR_KEY') }}"
+
+  tasks:
+  - name: Fail if 'CLUSTER_VERSION' is not defined
+    fail:
+      msg: "'CLUSTER_VERSION' must be a defined environment variable"
+    when:
+    - cluster_version == ''
+
+  - name: Set release_version to payload version
+    set_fact:
+      release_version: "{{ cluster_version | regex_search('^\\d+\\.\\d+') }}"
+
+  - name: Wait for host connection to ensure SSH has started
+    wait_for_connection:
+      timeout: 600
+
+  - name: Copy Atomic OpenShift yum repository certificate and key
+    copy:
+      src: "{{ ops_mirror_path }}"
+      dest: /var/lib/yum/
+
+  - name: Create rhel-7-server-ose-rpms repo file
+    template:
+      src: rhel-7-server-ose-4.X-devel-rpms.repo.j2
+      dest: /etc/yum.repos.d/rhel-7-server-ose-rpms.repo
+
+  - name: Create rhel-7-server-rpms repo file
+    copy:
+      src: rhel-7-server-rpms-4.X.repo
+      dest: /etc/yum.repos.d/
+EOF
+
+cat > rhel-7-server-ose-4.X-devel-rpms.repo.j2 <<-'EOF'
+[rhel-7-server-ose-{{ release_version }}-devel-rpms]
+name = A repository of dependencies for Atomic OpenShift {{ release_version }}
+baseurl = https://mirror.openshift.com/enterprise/reposync/{{ release_version }}/rhel-server-ose-rpms/
+failovermethod = priority
+gpgcheck = 0
+sslclientcert = /var/lib/yum/ops-mirror.pem
+sslclientkey = /var/lib/yum/ops-mirror.pem
+sslverify = 0
+enabled = 1
+EOF
+
+cat > rhel-7-server-rpms-4.X.repo <<-'EOF'
+[rhel-7-server-rpms]
+name = Red Hat Enterprise Linux 7 Server (RPMs)
+baseurl = https://mirror.openshift.com/enterprise/reposync/ci-deps/rhel-server-rpms/
+failovermethod = priority
+gpgcheck = 0
+sslclientcert = /var/lib/yum/ops-mirror.pem
+sslclientkey = /var/lib/yum/ops-mirror.pem
+sslverify = 0
+enabled = 1
+
+[rhel-7-server-optional-rpms]
+name = Red Hat Enterprise Linux 7 Server - Optional (RPMs)
+baseurl = https://mirror.openshift.com/enterprise/reposync/ci-deps/rhel-server-optional-rpms/
+failovermethod = priority
+gpgcheck = 0
+sslclientcert = /var/lib/yum/ops-mirror.pem
+sslclientkey = /var/lib/yum/ops-mirror.pem
+sslverify = 0
+enabled = 1
+
+[rhel-7-server-extras-rpms]
+name = Red Hat Enterprise Linux 7 Server - Extras (RPMs)
+baseurl = https://mirror.openshift.com/enterprise/reposync/ci-deps/rhel-server-extras-rpms/
+failovermethod = priority
+gpgcheck = 0
+sslclientcert = /var/lib/yum/ops-mirror.pem
+sslclientkey = /var/lib/yum/ops-mirror.pem
+sslverify = 0
+enabled = 1
+EOF
+
+# Dump the Ansible inventory for debugging
+ansible-inventory -i "${SHARED_DIR}/ansible-hosts" --list --yaml
+
+ansible-playbook -i "${SHARED_DIR}/ansible-hosts" prep.yaml -vvv
```

Comment:
shared_dir is for sharing content between steps, while the artifacts_dir (not the actual name) is for debugging information..

Reply:
The `ansible-inventory` command is just dumping the inventory text to the log.  I can redirect it to a file on the artifacts_dir if that is more appropriate.  I was using this for ensuring the inventory file was consistent between the provision and prep steps.

Reply:
I've updated this to redirect the created file to artifact_dir.